code/game/objects/items/weapons/soap.dm

```diff
@@ -54,7 +57,10 @@
 
 /obj/item/soap/attack(mob/target as mob, mob/user as mob)
 	if(target && user && ishuman(target) && ishuman(user) && !target.stat && !user.stat && user.zone_selected == "mouth" )
-		user.visible_message("<span class='warning'>\the [user] washes \the [target]'s mouth out with [name]!</span>")
+		user.visible_message("<span class='warning'>\the [user] starts washing \the [target]'s mouth out with [name]!</span>")
```

Comment:
I'm pretty sure these `\the` bits aren't encouraged anymore, the reference handles itself depending on proper/improper state of word